applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -17,6 +17,7 @@ class StabilizedFormulation(object):
     def __init__(self,settings):
         self.element_name = None
         self.condition_name = "MonolithicWallCondition"
+
```

Comment:
please do not add only newline changes
this messes up the git-history, which makes it harder to trace changes when debugging

Reply:
I hope I could solve this be copying the original file from the repository again. I apologize for committing these unnecessary changes and will take better care in the future.